### DISCUSSION TITLE
Possible to change max framerate?

### DISCUSSION THREAD

**Discussion by DesktopECHO** [neutrinolabs/xrdp#2953] [Q&A] [answered]
Hi, I have two idenical boxes running different builds of xrdp.  No hardware acceleration.
One is running the @Nexarian mainline_merge, the other [v0.10](https://github.com/neutrinolabs/xrdp/tree/v0.10) 

Both indicate:
`````[INFO ] xrdp_encoder_create: starting gfx rfx pro codec session`````

But v0.10 has much choppier video playback.  I notice xrandr reports back a refresh rate of 50 in v0.10: 

`````
$ xrandr
rdp0 connected 2560x1600+0+0 0mm x 0mm
   2560x1600     50.00* 
`````
Nexarian's mainline_merge has this set at `````25`````.  

Is there any way I can set this back to 25 in v0.10?  I suspect this highter refresh is overloading the CPU.  I've seen references online to this change in mainline_merge but I can't find the commit where's it made.   

Alternativley, is there a fix available in mainline_merge for RDP window minimizing in MSTSC.EXE?  Minimize the xrdp window, when it's un-minimized, the sesson window appears frozen.  The session is still alive -- Input and sound still work, it just stops painting the screen after it's minimized.

**Comment by Nexarian** [comment, marked as answer]
The difference in speed you're seeing is very likely due to H264 versus RFX Pro. H264 is just faster, but it's not in v0.10 yet.

`mainline_merge` I can update now that v0.10 is stable. It will be a whole-cloth rebuild rather than a rebase though, because so much has changed in the base branch.

**Reply by DesktopECHO** [reply]
Oops I thought H264 made it into v0.10, thanks for straightening that out.  
I will keep an eye out for your next build, good luck and thanks for the quick reply.

**Reply by Nexarian** [reply]
Getting RFX Pro was hard enough! Hopefully the foundation we've built over the last few months will make H264 easier, but each codec has its own mysteries and foibles.
